dimagi/pages/views/core.py

```diff
@@ -15,7 +15,25 @@
 
 from dimagi.utils.wordpress_api import get_json
 from dimagi.pages.models.partners import latestPartners
+from dimagi.pages.models.blog import BlogPost
+
+from dimagi.data.blog import (
+    get_category_by_slug,
+    nav_categories
+)
+
+def _get_posts(category, page=None, num_posts=None):
```

Comment:
duplicating code should never be the right answer when importing private methods. the better answer is to address why you needed call these methods directly inside of `views.core`

Reply:
Well I need to pull posts tagged with COVID. This logic is contained within these two methods. I can also export these two methods into some utility class and then I can include them in both files. Should I do that? Sorry for making some bad moves, I am trying to do multiple things at once as we are trying to rush this new page. I'll fix other tab/space related issues as well.